src/api/telemetry/TelemetryAPI.js

```diff
@@ -420,6 +420,22 @@ define([
         return this.valueFormatterCache.get(valueMetadata);
     };
 
+    /**
+     * Get a value formatter for a given valueMetadata.
+     * @param {string} key
+     *
+     * @returns {Format}
+     */
+    TelemetryAPI.prototype.getFormatter = function (key) {
+        if (!this.formatService) {
```

Comment:
Let's put this in a `getFormatService()` function so that we don't have to keep doing this.